package.json

```diff
@@ -35,7 +35,7 @@
     "lint:stylelint": "stylelint './src/**/*.scss' --syntax scss --ignorePath .gitignore",
     "prepare": "yarn build",
     "publish-npm": "yarn semantic-release",
-    "start": "yarn test:engines && yarn storybook",
+    "start": "yarn test:engines && NODE_OPTIONS='--max-old-space-size=16384' yarn storybook",
```

Comment:
Could you try a `killall node -9` and then try running this command?

I assume you saw a heap out of memory error and added this. These are typically a result of resource issues on the local machine vs a reproducible issue across other machines. We shouldn't set this flag unless necessary.

Reply:
yeah, it's fine if you wanna take it out, but I couldn't even get my initial build to work without it, and as far as I could tell I didn't have anything else eating local memory

Reply:
no luck with the `killall node -9` even after reboot, I couldn't do the `yarn start` without upping the heapsize!  not sure if something is strange with the new carbon.